model/project_ref.go

```diff
@@ -719,6 +719,16 @@ func (p *ProjectRef) AttachToNewRepo(u *user.DBUser) error {
 	if err != nil {
 		return errors.Wrap(err, "updating owner/repo in the DB")
 	}
+	_, err = EnableWebhooks(ctx, p)
+	if err != nil {
+		grip.Debug(message.WrapError(err, message.Fields{
+			"message":            "error enabling webhooks",
```

Comment:
`enabling webhooks` :)

Reply:
This only applies to `errors.Wrap` and `errors.Wrapf`. For `message.Fields`, it doesn't apply because it's a log message, not a wrapped error.